ui/src/utils/math-utils.ts

```diff
@@ -92,3 +92,22 @@ export function interpolateXY(point1: LayoutPoint, point2: LayoutPoint, mValue:
 function interpolate(value1: number, value2: number, portion: number): number {
     return value1 + (value2 - value1) * portion;
 }
+
+function sqr(x: number) {
+    return x * x;
+}
+
+function dist2(v: Point, w: Point) {
+    return sqr(v.x - w.x) + sqr(v.y - w.y);
+}
+
+export function distToSegmentSquared(p: Point, start: Point, end: Point) {
```

Comment:
Käytetäänkö tätä jossain? Näyttää välivaihe-funkkarilta joka voisi olla not exported.

Reply:
Toisaalta jos ei käytetä niin tuo sqrt:n voisi vain tehdä tässä ja sitten se on vain distance